_content/setup-wallets-programmatically.md

```diff
@@ -0,0 +1,148 @@
+## Set up and use Interledger accounts programmaticallly
+
+To get started using Interledger, this tutorial walks through using a test network programmatically. A test network uses fake money, rather than transferring real value. This Interledger testnet is currently maintained by teams at [Xpring](https://xpring.io/) and [Coil](https://coil.com/), but anyone can join.
+
+The Xpring and Rafiki testnet connectors, both used in this tutorial, together form a network for users to practice sending and receiving test currency values. Although this tutorial uses test XRP for simplicity, Interledger can handle transactions involving any currency pairs, as described in [Interledger Overview](overview.html).
+
+You will:
+
+1. [Create an account](#1-create-an-account) using the Xpring connector to the ILP testnet.
+2. [Add test XRP](#2-add-test-xrp-to-your-account) to your account using the Xpring "rainmaker", which is a "faucet" for dispensing test funds.
+3. [Check your account balance](#3-check-your-account-balance).
+4. [Send test XRP](#4-send-test-xrp-to-another-payment-pointer) to another payment pointer.
+5. [Receive test XRP](#5-receive-test-xrp).
+
+To do the same with a GUI-based interface, refer to [Set Up Interledger accounts with a user interface](setup-wallets.html).
+
+### 1. Create an account
+
+To create a new account, you send an HTTP request. For this tutorial, we'll use curl on the command line to make requests to the API, but you can use an HTTP client library in any language. Here's how to create an account on the command line.
+
+```
+> curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts' \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--data-raw '{
+  "assetCode": "XRP",
+  "assetScale": "6"
+}'
+```
+
+This request returns:
+- A payload that contains your `accountId`
+- An auth token which you use to authenticate to the connector
+- A [payment pointer](https://paymentpointers.org/) that can be used to receive value in your test account.
+
+Here's an example response payload for reference. Keep track of the generated `accountId` at `accountId`, and your generated auth token at `"customSettings"` -> `"ilpOverHttp.incoming.simple.auth_token"`, for use in subsequent steps.
+
+```text
+{
+    "accountId": "user_qqGV8nij",
+    "accountRelationship": "CHILD",
+    "assetCode": "XRP",
+    "assetScale": "6",
+    "maximumPacketAmount": null,
+    "linkType": {},
```

Comment:
small nit - `linkType` isn't being serialized properly currently (https://github.com/xpring-eng/hermes-ilp/issues/8).  Once that is fixed, it should read `"linkType": ILP_OVER_HTTP`.  

Not sure if we want to fake this and replace `linkType: {}` with `linkType: ILP_OVER_HTTP`.  It's not really pertinent information, especially for a Getting Started tutorial, but might raise some eyebrows...